app/dashboard/views.py

```diff
@@ -1468,7 +1468,7 @@ def new_bounty(request):
         confirm_time_minutes_target=confirm_time_minutes_target,
         active='submit_bounty',
         title=_('Create Funded Issue'),
-        update=bounty_params,
+        update=bounty_params
```

Comment:
This trailing `,` is fine and intentional, in the event someone adds a new line to `L1472` for another kwarg.

Reply:
Done